db/schema.rb

```diff
@@ -801,6 +802,7 @@
     t.string "template"
     t.string "title"
     t.datetime "updated_at", null: false
+    t.boolean "use_partial", default: false
```

Comment:
I bet everyone's gonna hate this 😆 

Since this is just for development, perhaps we could just use a partial if one if found at a location like `app/views/pages/#{page.slug}.html.erb`?

Reply:
I can see the point and its a very valid one, to make sure nothing gets into the DB that doesn't really need to be there. Also that column name is just looks terrible 🙃

Automatically recognizing the presence of a partial that matches the slug sounds interesting (the type of "magic" that comes out of the box in Rails). I can give this approach a try to see how it feels.

Reply:
I did something similar here, just having a method construct the name instead of storing it in the db: https://github.com/thepracticaldev/dev.to/pull/8149/files#diff-8135377fbb811479614b50a9eca1d721R21-R23

Reply:
Noted! Thanks for the reference, definitely useful 😄